recipes/mfast/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class mFASTConan(ConanFile):
+    name = "mfast"
+    license = "LGPL-3.0"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

https://github.com/objectcomputing/mFAST/blob/master/licence.txt

Reply:
Ah! They changed the  license at some point. Version 1.2.1 I think is still LGPLv3:

https://github.com/objectcomputing/mFAST/blob/v1.2.1/licence.txt

Is it better to leave it as it is now or to update already the license to match the most updated one?

Reply:
Indeed. Keep LGPL-3.0, for future versions we can use dynamic license.